isic/ingest/services/publish/__init__.py

```diff
@@ -94,7 +101,62 @@ def cohort_publish(*, publish_request: PublishRequest) -> None:
     sync_elasticsearch_indices_task.delay_on_commit()
 
 
+def embed_iptc_metadata(blob: FieldFile, attribution: str, license_: str) -> Path:
+    # pyexiv2 operates on filenames directly, so we need to write to a temp file
+    with tempfile.NamedTemporaryFile(delete=False) as temp_file:
+        shutil.copyfileobj(blob.file, temp_file)
+        temp_file.flush()
+
+        with pyexiv2.Image(temp_file.name) as tmp_image:
+            tmp_image.modify_iptc(
+                {
+                    "Iptc.Application2.Copyright": "Copyright Creative Commons",
```

Comment:
The copyright isn't held by Creative Commons.

Could you include a link in the PR description to the IPTC field definitions?

Reply:
- https://developers.google.com/search/docs/appearance/structured-data/image-license-metadata#iptc-photo-metadata
- https://www.iptc.org/std/photometadata/specification/IPTC-PhotoMetadata#metadata-properties

Reply:
What should we put, if anything?